examples/pytorch/ogb/sign/README.md

```diff
@@ -0,0 +1,49 @@
+SIGN: Scalable Inception Graph Neural Network
+==========================
+Paper: [https://arxiv.org/abs/2004.11198](https://arxiv.org/abs/2004.11198)
+
+
+Dependencies
+------------
+- pytorch 1.5
+- dgl 0.5 (tested with 0.6a201020 nightly build)
+- ogb 1.2.3
+
+
+How to Rrun
```

Comment:
Run

Reply:
Fixed. I also updated the readme in ogb folder